1-js/05-data-types/07-map-set/article.md

```diff
@@ -158,7 +157,7 @@ let map = new Map([
 alert( map.get('1') ); // str1
 ```
 
-如果我们有个纯对象, 并且想利用这个纯对象来创建 `Map`, 可以使用内建方法 [Object.entries(obj)](mdn:js/Object/entries)，它返回一个具有相同格式的并且带有键值对的数组对象。
+如果我们想从一个已有的普通对象来创建 `Map`，可以使用内置方法 [Object.entries(obj)](mdn:js/Object/entries)，它返回一个该类型对象的键值对数组 //TODO。
```

Comment:
@KerbalHao 惭愧忘记了我标了 TODO 的这个地方，感谢翻译，我改一下

Reply:
@KerbalHao 

>网上对于 [plain object] 好像都翻译为 [简单对象]
我查了下 MDN，好像没有 plain object 的对应翻译，你这边一般查的是哪里呢？

这个”简单对象“我感觉会比较有歧义，因为有简单就会有复杂，会造成还有复杂对象的错觉，所以我用了普通对象。不过我 js 不熟，这里的上下文是否内含了简单和复杂的意思呢？

> [普通对象] -> [简单对象]，我查了一下，网上对于 [plain object] 好像都翻译为 [简单对象]，我觉得这里可以这样写

Reply:
@KerbalHao 不好意思更正一下，感觉还是你说的”简单对象“对着呢。

我查了一下 [1-js/08-prototypes/04-prototype-methods/article.md)，这里似乎解释了, regular plain object，看来 plain 还是和 regular 有区别的。

> We can call such objects "very plain" or "pure dictionary" objects, because they are even simpler than the regular plain object {...}.

不过现在其他 article 中也有类似的 plain object,大概看了下，有译作简单对象的，也有译作普通对象的，不太统一，请教@leviding？

Reply:
@leviding 大佬，有空的话，麻烦看看这边？

Reply:
@KerbalHao @bemself 这种没有统一翻译的，翻译了会有歧义的，就全用英文？